src/utils/parsers/custom.ts

```diff
@@ -13,9 +13,15 @@ export class CustomParser extends BaseParser {
             throw new Error(`No match pattern for ${extension}`)
         }
 
+        this.identity = `custom ${extension}`
```

Comment:
aren't you overwriting the identity field being set to "custom" above? Is that going to cause problems elsewhere? We use the identity to figure out which parser to use don't we

Reply:
Yeah, the identity "custom" isn't unique, but we create a custom parser for each extension. This was an issue when the `show-regex` flag is enabled and we're printing values to the console, we build a map to track which parsers have been printed, and since all custom parsers have the same identity only one of them would be logged

Reply:
maybe just remove the line that's assigning it to `custom` then since that value should never be used anyway?